src/utils/css/pseudo.ts

```diff
@@ -13,7 +13,7 @@ type absolutePseudos = 'before' | 'after';
 
 export const absolute = (pseudo: absolutePseudos) => css`
   position: relative;
-  &:${/* sc-prop */ pseudo} {
+  &: ${/* sc-prop */ pseudo} {
```

Comment:
Does this change affect behaviour? It was automatically applied by prettier after update.

Reply:
The behaviour does indeed change. This is probably a prettier bug (https://github.com/prettier/prettier/pull/6794). So added a prettier-ignore rule for this block. Can be removed once bug is fixed and functionality tested again.